docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+   app:
+       build:
+           context: .
+       ports:
+           - "8001:8000"
+       volumes:
+           - ./:/app
+           - my_media:/files/media
+       command: >
+           sh -c "python manage.py wait_for_db &&
+                  python manage.py migrate &&
+                  python manage.py runserver 0.0.0.0:8000"
+       env_file:
+           - .env
+       depends_on:
+           - db
```

Comment:
The `depends_on` feature only ensures that the `db` service is started before the `app` service, but it does not wait for the database to be ready to accept connections. Consider using a script or a tool like `wait-for-it` to ensure the database is ready before running migrations and starting the server.